docs/asciidoc/import/loadjson.adoc

```diff
@@ -98,8 +98,8 @@ image::apoc.load.json.so.png[scaledwidth="100%"]
 [source,cypher]
 ----
 WITH "https://api.stackexchange.com/2.2/questions?pagesize=100&order=desc&sort=creation&tagged=neo4j&site=stackoverflow&filter=!5-i6Zw8Y)4W7vpy91PMYsKM-k9yzEsSC1_Uxlf" AS url
-CALL apoc.load.json(url,'$.items.owner.name') YIELD value
```

Comment:
should we add this as a new example instead?

Should the old one not still work?